### ISSUE TITLE
Couldn't select default buffer in `bufler-workspace-switch-buffer` 

### ISSUE THREAD

**Issue by sarg** [alphapapa/bufler.el#30]
The feature done in #23 is not working for me. When I open `bufler-workspace-switch-buffer` it shows the `other-buffer` at the top. The name of the buffer is correct, but the path is missing.
But I couldn't switch to that buffer as I get `eq: Wrong type argument: stringp, nil`. I think it happens because you get `selected-buffer` with `(alist-get ... #'string=)` and it produces `nil`.

Here is what I see:
```
24   Buffer:   
bufler.el
X11 » WWW » [github.com] New Issue · alphapapa/bufler.el
Projectile: bufler.el » bufler-workspace.el
Dir: /home/sarg/ » *scratch*
```
The first line should be `Projectile: bufler.el » bufler.el` instead.

**Comment by alphapapa** [comment]
There may be a bug here, but I can't reproduce it.  Please use `emacs -q` or `emacs-sandbox.sh` and provide a specific, step-by-step guide to reproduce it.

**Comment by sarg** [comment]
You're right, the bug in a hook that doom-emacs provides. On my setup `(switch-to-buffer nil)` doesnt' work and I'll report it to doom's authors.

But anyway, bufler shows `(other-buffer)` twice - one on top, just the buffer name and another in the list with proper path.

**Comment by sarg** [comment]
Doom has fixed the issue: https://github.com/hlissner/doom-emacs/issues/2992

**Comment by alphapapa** [comment]
> Doom has fixed the issue: hlissner/doom-emacs#2992

Thanks for the update.

> You're right, the bug in a hook that doom-emacs provides. On my setup (switch-to-buffer nil) doesnt' work and I'll report it to doom's authors.

Since it's not a bug in this package, I'll close it.

> But anyway, bufler shows (other-buffer) twice - one on top, just the buffer name and another in the list with proper path.

In the sample you showed, I don't see that buffer in the list twice.  If this still appears to be a bug, would you file it separately please?
